diff --git a/core/modules/system/system.install b/core/modules/system/system.install
index 2b626c5..83c9abd 100644
--- a/core/modules/system/system.install
+++ b/core/modules/system/system.install
@@ -46,7 +46,7 @@ function system_requirements($phase) {
     // is not running the default installation profile.
     $profile = drupal_get_profile();
     if ($profile != 'standard') {
-      $info = system_get_info('module', $profile);
+      $info = system_get_info('profile', $profile);
       $requirements['install_profile'] = [
         'title' => t('Installation profile'),
         'value' => t('%profile_name (%profile-%version)', [
